### PR TITLE
Beta to develop

### LINKED ISSUE #1356: Post deployment actions after 1.3.4 release
Need to update main branches after 1.3.4 version was released.

Following the steps described here:
https://github.com/skalenetwork/internal-support/blob/main/docs/wiki/smart-contracts-development/release-process.md

### PR DESCRIPTION
**Pull request by DimaStebaev**

### CODE REVIEW

**Review by yavrsky — DISMISSED** [review]

File: proxy/package.json [hunk 1]
```diff
@@ -26,8 +26,8 @@
         "@openzeppelin/contracts-upgradeable": "^4.7.1",
         "@openzeppelin/hardhat-upgrades": "^1.9.0",
         "@skalenetwork/etherbase-interfaces": "^0.0.1-develop.20",
-        "@skalenetwork/ima-interfaces": "1.0.0-develop.24",
-        "@skalenetwork/skale-manager-interfaces": "1.0.0",
+        "@skalenetwork/ima-interfaces": "^1.1.0-develop.0",
```
Comment (payvint) [inline proxy/package.json:29]: Looks like we can use stable version here

File: proxy/package.json [hunk 2]
```diff
@@ -26,8 +26,8 @@
...
+        "@skalenetwork/skale-manager-interfaces": "1.0.0-develop.1",
```
Comment (payvint) [inline proxy/package.json:30]: The same here

File: proxy/package.json (hunk 1 above)
Comment (DimaStebaev) [reply proxy/package.json:29]: Initially I tried it but we already have interface changes in `develop`.

File: proxy/package.json (hunk 2 above)
Comment (DimaStebaev) [reply proxy/package.json:30]: We have not released a new stable version of  `skale-manager-interfaces` yet.

File: proxy/migrations/tools/verification.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ export async function verify(contractName: string, contractAddress: string, cons
                 });
                 break;
             } catch (e: any) {
-                if (e.toString().includes("Already Verified")) {
+                if (e.toString().includes("Contract source code already verified")) {
```
Comment (payvint) [inline proxy/migrations/tools/verification.ts:15]: Let's change it in the next PR to develop like `e.toString().toLoweCase().includes("Already Verified")`

**Review by payvint — APPROVED** [review]

**Review by sergiy-skalelabs — APPROVED** [review]

**Pull request merged**